File: app/src/main/java/com/BlizzardArmory/warcraft/equipment/InventoryType.java
package com.BlizzardArmory.warcraft.equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class InventoryType {

    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("name")
    @Expose
    private String name;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/InventoryType.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Inventory type.
 */
@Keep
data class InventoryType(

    @SerializedName("type")
    var type: String,

    @SerializedName("name")
    var name: String

)